commit c7a74029969b912241591b2889961b17514ca61f
Author: Andy Whitcroft <apw@canonical.com>
Date:   Fri Jun 26 10:59:30 2009 +0100

    UBUNTU: SAUCE: disable adding scsi headers to linux-libc-dev
    
    Currently scsi headers are generated by the kernel and by libc6-dev.
    We need to coordinate any switch over to the kernel.  Temporarily disabled
    these headers in the kernel package.
    
    Signed-off-by: Andy Whitcroft <apw@canonical.com>

diff --git a/include/Kbuild b/include/Kbuild
index 8d226bf..fe36acc 100644
--- a/include/Kbuild
+++ b/include/Kbuild
@@ -9,4 +9,3 @@ header-y += rdma/
 header-y += video/
 header-y += drm/
 header-y += xen/
-header-y += scsi/
diff --git a/include/scsi/Kbuild b/include/scsi/Kbuild
deleted file mode 100644
index f2b9491..0000000
--- a/include/scsi/Kbuild
+++ /dev/null
@@ -1,4 +0,0 @@
-header-y += scsi_netlink.h
-header-y += scsi_netlink_fc.h
-header-y += scsi_bsg_fc.h
-header-y += fc/
